src/runtime/routes/openapi.ts

```diff
@@ -96,3 +115,9 @@ function defaultTags(route: string) {
 
   return tags;
 }
+
+export function defineOpenAPISchema(schema: NitroOpenapiSchema) {
+  useStorage(schema.routeBase).setItem(`openapi-${schema.method}`, schema);
```

Comment:
This is a really interesting approach however considering overhead and limitations (dev<>build communication) I think we have to use static code extraction for route meta data.

Reply:
I changed the approach to parsing it at build time, please check if this version seems to be on the right path at least

Reply:
@pi0 I redid the parse to build time, could you please check if this is better? Got some inspiration from [(Nuxt's extract route rules)](https://github.com/nuxt/nuxt/blob/95a521376614d9cf605ca16fefe80d6c216bd906/packages/nuxt/src/pages/route-rules.ts#L25)